scripts/debug_tools/prepare_all_cmd_for_ctu.py

```diff
@@ -35,6 +36,18 @@ def execute(cmd):
         raise
 
 
+def get_triple_arch(analyze_command_file):
+    with open(analyze_command_file) as f:
+        cmd = f.readline()
+
+    cmd = cmd.split()
+    for flag in cmd:
+        if flag.startswith('--target='):
+            return flag[9:].split('-')[0]  # 9 == len('--target=')
+
+    return platform.machine()
```

Comment:
`platform.machine()` would return the same string what clang uses when `--target` is not given?

Reply:
Debatable. What happens if the architecture of Python and Clang differs, exactly one of them is 32-bit?

Reply:
This is just a fallback in case there is no `--target` given in the `clang` command invocation. This `platform.machine()` returns the architecture of the current machine where this Python script is run. If it differs from the one on which the build action was run, then it still doesn't work, like before this patch.

Reply:
Ok, let's merge it now because we need this badly and most often `--target` is given.